### DISCUSSION TITLE
Cannot set count in videos-function

### DISCUSSION THREAD

**Discussion by UzhFAN** [davidteather/TikTok-Api#897] [Q&A] [answered]
Dear David,

Thank you for this amazing project. I'm wanting to use it to study political interactions of teens in Switzerland. Right now however, I'm already slightly struggling to understand how to set the number of videos collected. I seem to not be able to change it form the default number of count=30.

Super simple question, but help would be greatly appreciated.
Yours,
Nicole

**Comment by davidteather** [comment, marked as answer]
Sorry but the repository is sort of in a broken state right now, not sure when/how I will fix it.

**Reply by davidteather** [reply]
Fixed!
